src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -68,7 +68,8 @@ azure-mgmt-recoveryservicesbackup==5.1.0
 azure-mgmt-redhatopenshift==1.2.0
 azure-mgmt-redis==14.1.0
 azure-mgmt-relay==0.1.0
-azure-mgmt-resource==22.0.0
+azure-mgmt-reservations==2.0.0
+azure-mgmt-resource==21.2.0
```

Comment:
The SDK package `azure-mgmt-resource` upgraded to `22.0.0` in PR #25434. Could you please pull the latest code from remote dev branch?